dotCMS/src/main/java/com/dotmarketing/viewtools/WorkflowTool.java

```diff
@@ -97,10 +98,18 @@ public WorkflowAction findAction(String id, User user) throws DotDataException,
 		return wapi.findActions(step, user);
 	}
 
+	public List<WorkflowAction> findActions(WorkflowStep step, User user, Permissionable permissionable) throws DotDataException, DotSecurityException {
+		return wapi.findActions(step, user, permissionable);
+	}
+
 	public List<WorkflowAction> findActions(List<WorkflowStep> steps, User user) throws DotDataException, DotSecurityException {
 		return wapi.findActions(steps, user);
 	}
 
+	public List<WorkflowAction> findActions(List<WorkflowStep> steps, User user, Permissionable permissionable) throws DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 156 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)